app/Http/Controllers/ReservationController.php

```diff
@@ -29,22 +28,89 @@ public function viewReservationOverview() {
     }
 
     public function handleCreateReservation(Request $request) {
-        $validated = $request->validate([
-            'contact' => 'required',
-            'arrival' => 'required',
-            'departure' => 'required',
-        ]);
+        $this->validateCreateReservation($request);
+
+        $invoice_id = $this->handleReservationInvoice();
+
+        $contact = $this->handleReservationContact($request);
+
+        $reservation = $this->storeReservation($request, $invoice_id, $contact);
+
+        $contact->reservations()->save($reservation);
```

Comment:
hier zou ik waarschijnlijk een nieuwe method in contact model voor maken, en reservation eraan meegeven